### ISSUE TITLE
`docs/conf.py` references missing debian changelog

### ISSUE THREAD

**Issue by dviererbe** [canonical/ubuntu-packaging-guide#11]
The `docs/conf.py` file references the debian changelog file we removed with PR #8.

```python
changelog = "debian/changelog"
if os.path.exists(changelog):
    changelogfile = codecs.open(changelog, mode='r', encoding='utf-8')
    head = changelogfile.readline()
    match = re.compile(".*\((.*)\).*").match(head)
    if match:
        version = match.group(1)
    changelogfile.close()
```

Note: This is no critical problem as the code checks if the file is present before using it.

**Comment by s-makin** [comment]
If the debian folder was there, I thought it might be necessary to keep this - but since we've moved it to another branch, I guess we don't need this after all. I can remove it, if you agree it's not needed.

**Comment by dviererbe** [comment]
As far as I can tell, this code does nothing. I don't see any variable used elsewhere. Yesterday, I thought that the version/changelog information was used somewhere.

**TL;DR:** I think we can just delete it.

**Comment by s-makin** [comment]
[Created this PR](https://github.com/canonical/ubuntu-packaging-guide/pull/12) to remove it
